core/services/chainlink/testdata/config-full.toml

```diff
@@ -189,19 +195,21 @@ FlagsContractAddress = '0xae4E781a6218A8031764928E88d457937A954fC3'
 LinkContractAddress = '0x538aAaB4ea120b2bC2fe5D296852D948F07D849e'
 LogBackfillBatchSize = 17
 LogPollInterval = '1m0s'
-MaxInFlightTransactions = 19
-MaxQueuedTransactions = 99
 MinIncomingConfirmations = 13
-MinimumContractPayment = '9.223372036854775807 link'
+MinContractPayment = '9.223372036854775807 link'
 NonceAutoSync = true
 NoNewHeadsThreshold = '1m0s'
 OperatorFactoryAddress = '0xa5B85635Be42F21f94F28034B7DA440EeFF0F418'
 RPCDefaultBatchSize = 17
 RPCBlockQueryDelay = 10
-TxReaperInterval = '1m0s'
-TxReaperThreshold = '1m0s'
-TxResendAfterThreshold = '1h0m0s'
-UseForwarders = true
+
+[EVM.Tx] # TODO Txes? Transactions?
```

Comment:
If we group them all together like this, them maybe spelling out the full word makes more sense :shrug: